templates/templates/preview/clientconfig_plain_text.template

```diff
@@ -0,0 +1,51 @@
+   
+        Client Configuration:
+            {%- if manifest.dbs %}
+            {%- if manifest.dbs[0].name %}
+            Database identifier: {{ manifest.dbs[0].name }}
```

Comment:
is hardcoded 0-th index ok?

Reply:
did some refactor of how client configuration was shown for each entity